styles/variables.scss

```diff
@@ -194,6 +194,24 @@ $tooltip-bg: $white;
 $tooltip-color: $light-gray;
 $tooltip-arrow-color: $white;
 
+// Breakpoints
```

Comment:
double-check that this values aren't supposed to come _after_ the variables import

Reply:
@moshmage it has to come before the variables import
[Bootstrap Sass Importing](https://getbootstrap.com/docs/5.0/customize/sass/#:~:text=//%202.%20Include%20any%20default%20variable%20overrides%20here)